README.md

```diff
@@ -43,15 +43,17 @@ Debugging on iOS directly from XCode is possible with a plugin (https://github.c
 
 # Android #
 
-Abacus builds and pushes the JVM target as a Github packagewith the followinng command:
+Abacus builds and pushes the JVM target as a Github package with the following command:
 
 > ./publish_android.sh
 
 The Android app (https://github.com/dydxprotocol/native-android) has the Gradle build step to pull the Abacus target from GithubPackage.
 
 # Web #
 
-Abaacus generates a Javascript / Typescript UMD module with the following command:
+## If you are using v4-abacus with the v4-web repo, follow local development instructions [here.](https://github.com/dydxprotocol/v4-web?tab=readme-ov-file#local-abacus-development) 
```

Comment:
personal pet peeve, can we remove the `.` from the url linking lmao (i.e. `[here](https://github.com/dydxprotocol/v4-web?tab=readme-ov-file#local-abacus-development).`)

Side note, do you think it'd be worth adding a comment that the linked development instructions should make the three commands below unnecessary?

Reply:
Yeah good callout, I'll make that more clear